### PR TITLE
init orbidder adapter

### PR DESCRIPTION
**Pull request by siggi-otto**

### CODE REVIEW

File: exchange/adapter_map.go [hunk 1]
```diff
@@ -79,6 +80,7 @@ func newAdapterMap(client *http.Client, cfg *config.Configuration, infos adapter
 		openrtb_ext.BidderGumGum:     gumgum.NewGumGumBidder(cfg.Adapters[string(openrtb_ext.BidderGumGum)].Endpoint),
 		openrtb_ext.BidderLockerDome: lockerdome.NewLockerDomeBidder(cfg.Adapters[string(openrtb_ext.BidderLockerDome)].Endpoint),
 		openrtb_ext.BidderOpenx:      openx.NewOpenxBidder(cfg.Adapters[string(openrtb_ext.BidderOpenx)].Endpoint),
+		openrtb_ext.BidderOrbidder: orbidder.NewOrbidderBidder(cfg.Adapters[string(openrtb_ext.BidderOrbidder)].Endpoint),
```
Comment (udojuettner6667) [inline exchange/adapter_map.go]: Identation broken.

File: adapters/orbidder/orbidder.go [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+	var bidResp openrtb.BidResponse
+	if err := json.Unmarshal(response.Body, &bidResp); err != nil {
+		return nil, []error{err}
+	}
+
+	bidResponse := adapters.NewBidderResponseWithBidsCapacity(5)
+
+	for _, sb := range bidResp.SeatBid {
+		for i := range sb.Bid {
```
Comment (udojuettner6667) [inline adapters/orbidder/orbidder.go]: ``
		for _, bid := range sb.Bid {
			bidResponse.Bids = append(bidResponse.Bids, &adapters.TypedBid{
				Bid:     &bid,
				BidType: openrtb_ext.BidTypeBanner,
			})
		}
``
... feels a bit more "natural" to me (ignoring the index, but use the bid object directly instead).

Comment (siggi-otto) [reply adapters/orbidder/orbidder.go]: was shamelessly copied from another bidder, but I am going to adjust that

File: adapters/orbidder/orbidder.go [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+		}}
+	}
+
+	var bidResp openrtb.BidResponse
+	if err := json.Unmarshal(response.Body, &bidResp); err != nil {
+		return nil, []error{err}
+	}
+
+	bidResponse := adapters.NewBidderResponseWithBidsCapacity(5)
+
```
Comment (arneschulz1984) [inline adapters/orbidder/orbidder.go]: Warum an dieser Stelle fest eine "5"?

**Pull request merged**